README.md

```diff
@@ -20,6 +20,8 @@ Repository | NIO 2 (Swift 5+) | NIO 1 (Swift 4+)
 [https://github.com/apple/swift-nio-transport-services][repo-nio-transport-services] <br> first-class support for macOS, iOS, tvOS, and watchOS | `from: "1.0.0"` | `from: "0.1.0"`
 [https://github.com/apple/swift-nio-ssh][repo-nio-ssh] <br> SSH support | `.upToNextMinor(from: "0.2.0")` | _n/a_
 
+NIO 2.29.0 and older support Swift 5.0+.
```

Comment:
Wondering if there is a better way of representing this.  5.0 would work if they did `from: 2.0.0`

Reply:
5.0 would work if they did `from`: anything 2.29.0 and earlier. I think this is probably sufficient.